Assets/MRTK/Tests/PlayModeTests/Experimental/BoundsControlTests.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.MixedReality.Toolkit.Tests.Experimental
     /// <summary>
     /// Tests for runtime behavior of bounds control
     /// </summary>
-    public class BoundsControlTests
+    public class PlayModeBoundsControlTests
```

Comment:
What's this for?

Reply:
Just a refactor because the clicking on the play mode boundscontroltests to open it would open up the edit mode boundscontroltests file instead.